src/client/App.css

```diff
@@ -122,36 +143,37 @@ h2 {
   grid-area: email;
 }
 
-#registrationForm .usernameLabel {
+.usernameLabel {
   grid-area: usernameLabel;
 }
-#registrationForm #username {
+#username {
   grid-area: username;
 }
 
-#registrationForm .passwordLabel {
+.passwordLabel {
   grid-area: passwordLabel;
 }
-#registrationForm #password {
+#password {
   grid-area: password;
   font-weight: 100;
 }
```

Comment:
I like how you used more global styles for forms so they can be reused

Reply:
Thank you :)